test/tensor/test_expression.cpp

```diff
@@ -0,0 +1,168 @@
+//  Copyright (c) 2018 Cem Bassoy
+//
+//  Distributed under the Boost Software License, Version 1.0. (See
+//  accompanying file LICENSE_1_0.txt or copy at
+//  http://www.boost.org/LICENSE_1_0.txt)
+//
+//  The authors gratefully acknowledge the support of
+//  Fraunhofer and Google in producing this work
+//  which started as a Google Summer of Code project.
+//
+
+
+
+
+#include <boost/numeric/ublas/tensor/expression.hpp>
+#include <boost/numeric/ublas/tensor/tensor.hpp>
+#include <boost/test/unit_test.hpp>
+#include "utility.hpp"
+
+#include <functional>
+#include <complex>
+
+
+
+using test_types = zip<int,long,float,double,std::complex<float>>::with_t<boost::numeric::ublas::first_order, boost::numeric::ublas::last_order>;
+
+
+struct fixture {
+	using extents_type = boost::numeric::ublas::shape;
+	fixture() : extents{
+								extents_type{},            // 0
```

Comment:
Same formatting (newlines before braces), plus remove excessive indentation.

Reply:
Will use clang-format now for that purpose.